packages/ui/src/components/EditBox.svelte

```diff
@@ -132,7 +132,7 @@
     focusManager?.setFocus(idx)
   }
   $: if (input) {
-    input.addEventListener('focus', updateFocus, { once: true })
+    input.addEventListener('focus', updateFocus)
```

Comment:
I'm not actually sure about this change. We use similar pattern in several places. Could you describe what problem does it fix?

Reply:
Using this pattern event listener is executed only once. So custom focus manager focused position gets updated only on first time focusing the field. So - click on field, tab or enter to next ones, click on field again - tabbing or entering continues from where it was, not from currently selected/focused field, which is counterintuitive.